src/VAC/Global.cpp

```diff
@@ -962,6 +965,36 @@ void Global::setPasteDelta(double delta)
     pasteDeltaY_ = delta;
 }
 
+const QRectF& Global::selectedGeometry() const
+{
+    return selectedGeometry_;
+}
```

Comment:
this and the other getter can be inline as the data is stored in the object without pimpl

Reply:
Done